dataflow/flex-templates/kafka_to_bigquery/pom.xml

```diff
@@ -31,7 +31,7 @@
     <beam.version>2.19.0</beam.version>
     <kafka.version>2.4.0</kafka.version>
 
-    <maven-enforcer-plugin.version>3.0.0-M3</maven-enforcer-plugin.version>
+    <maven-enforcer-plugin.version>3.0.0-SNAPSHOT</maven-enforcer-plugin.version>
```

Comment:
@kurtisvg or @averikitsch or @lesv 
Do we take `SNAPSHOT` versions?

Reply:
Not if we can help it.

Reply:
Kk, I'll close it and see if renovate opens a new one or they fixed their release.